tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -1558,7 +1565,7 @@ static SDKNode *constructTypeAliasNode(SDKContext &Ctx,TypeAliasDecl *TAD) {
 static void addMembersToRoot(SDKContext &Ctx, SDKNode *Root,
                              IterableDeclContext *Context) {
   for (auto *Member : Context->getMembers()) {
-    if (shouldIgnore(Member))
+    if (shouldIgnore(Member, const_cast<Decl*>(Context->getDecl())))
```

Comment:
You don't need to const_cast, just add the the parameter as `static bool shouldIgnore(Decl *D, const Decl* Parent)`